perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -45,11 +45,21 @@
 from perfkitbenchmarker import vm_util
 from perfkitbenchmarker.sample import PercentileCalculator  # noqa
 
+
 flags.DEFINE_enum('storage', benchmark_spec_class.GCP,
                   [benchmark_spec_class.GCP, benchmark_spec_class.AWS,
                    benchmark_spec_class.AZURE],
                   'storage provider (GCP/AZURE/AWS) to use.')
 
+flags.DEFINE_string('object_storage_gcp_region', 'us-central1',
+                    'Region for GCP object storage.')
+
+flags.DEFINE_string('object_storage_aws_region', 'us-east-1',
```

Comment:
I could. I made them three separate flags because it makes the default values clear and also because it lets me print a warning if the user passes the wrong flag for their storage provider. I'm open to merging them if you think it's more clear.